test/framework/sql_query.cpp

```diff
@@ -0,0 +1,297 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "framework/sql_query.hpp"
+
+#include <soci/boost-tuple.h>
+#include "ametsuchi/impl/soci_utils.hpp"
+#include "backend/protobuf/permissions.hpp"
+#include "common/bind.hpp"
+#include "common/result.hpp"
+#include "interfaces/common_objects/common_objects_factory.hpp"
```

Comment:
Already included in "framework/sql_query.hpp", so it is better to forward declare it there.